Modelica/Fluid/Examples/Explanatory.mo

```diff
@@ -267,4 +267,223 @@ The total pressures offer an additional perspective on the model. After setting
           "Plot the model results"),
       experiment(StopTime=1.1));
   end MomentumBalanceFittings;
+
+  model SimplerHeatingSystem
```

Comment:
The comparitive `simpler` here and in the description text leaves it open what's the reference.

Reply:
However, the info-layer starts with:
"
Simple heating system with a closed flow cycle.
It is based on <a href=\"Modelica.Fluid.Examples.HeatingSystem\">HeatingSystem</a>.
The only change is that pump is simpler removing a non-linear system of equations for N that could in some cases fail."

I think that it is the correct place for a more detailed explanation (can't really have that in the description), and that the text explains how it is "simpler", although it might be possible to improve it even further.